src/os/file_unix.go

```diff
@@ -223,3 +224,17 @@ func newUnixDirent(parent, name string, typ FileMode) (DirEntry, error) {
 	ude.info = info
 	return ude, nil
 }
+
+// Since internal/poll is not available, we need to stub this out.
+// Big go requires the option to add the fd to the polling system.
+//
+//go:linkname net_newUnixFile net.newUnixFile
+func net_newUnixFile(fd int, name string) *File {
```

Comment:
I agree with the need to stub this out but I'm concerned that the blocking/non-blocking requirement mentioned in the upstream Go file might bite us later.